app.js

```diff
@@ -206,6 +207,7 @@ app.get('/help/about', (req, res) =>
 )
 app.get('/map', (req, res) => res.redirect('https://streetmix.github.io/map/'))
 app.get('/survey', controllers.survey.get)
+app.post('/refresh', cors(), controllers.refresh.post)
```

Comment:
[I lost a draft of this in mid-write, will try to rewrite quickly, forgive brevity. Please ask questions if anything is unclear.]

Let's talk about paths. Each time we add a path at the root level, we potentially collide with the userland namspace. (What happens if we have a user named `map`?) To manage this, we have to keep track of each reserved url on the front end here: https://github.com/streetmix/streetmix/blob/2c1ca48110aa1b487fbc5d45001d93c871425e34/assets/scripts/app/constants.js#L38

As you can see we do not, but should, keep this up to date.

There are some things we can do to help improve this, which involves changing the back-end URL structure, and is outside of the scope of this PR. But I think we can start by not introducing new ones. (`/survey` was the last one I thought could be justifiable.) In this case `/refresh` is never meant to be linked to by the user, so there's no point in keeping this URL here, in a way that could collide with a potential user name.

Can we move this to `/services/auth/refresh-login-token`? This would be a little clearer and would be under a namespace which limits future namespace collisions. I'm aware this is much different from the structure other auth callbacks have taken, but I think we can decide on restructuring them as well.

Reply:
👌 makes sense, I hadn't realized this might collide with users. Definitely agree it's better to move this into something `/services/auth/refresh-login-token` since that's the case.